README.rst

```diff
@@ -9,70 +9,89 @@ Open Zaak
-.. _`Notificaties API`: https://zaakgerichtwerken.vng.cloud/standaard/notificaties/index
-.. _`Open Notificaties`: https://github.com/open-zaak/open-notificaties
+* Beheer: er is een beheerportaal ingericht waarmee de verschillende API's door functioneel beheerders kunnen worden beheerd.
+* Performance: er zijn performance-metingen verricht op basis van schattingen van de verwachte belasting door applicaties die eindgebruikers gebruiken. Benodigde verbeteringen zijn doorgevoerd waardoor een belasting door 2000 eindgebruikers geen problemen zou moeten opleveren.
+* Documentatie van de componenten, met name van de beheer applicaties. (De inhoudelijke documentatie over de API's is onderdeel van de standaard.)
+* Uitrol: Om de uitrol naar servers te vereenvoudigen zijn er Docker containers en een Virtual Machine (VM) image beschikbaar. Dit zijn een soort componenten die gemakkelijk kunnen worden uitgerold op een server om ze vervolgens in gebruik te nemen. Hiermee kunnen gemeenten de API’s op eenvoudige wijze (laten) draaien bij een hostingpartij.
 
-**Open Zaak** gebruikt de code van de
-`referentie implementaties van VNG Realisatie`_ als basis om een stabiele set
-API's te realiseren die in productie gebruikt kunnen worden bij gemeenten.
+Architectuur van OpenZaak
+===========
+
+De architectuur van OpenZaak is gebaseerd op een beperkt aantal componenten. De belangrijkste component is de registratiecomponent die de API's voor ZGW aanbiedt. Daarnaast zijn er de volgende componenten:
+
+* Notificatie-component die Notificaties API aanbiedt
+* Referentielijsten component die wordt gebruikt om de selectielijst voor archiveren te ontsluiten
+* Beheerportaal dat toegang biedt tot de verschillende beheerapps die bij de API's horen
+
+.. image:: assets/openzaak-component-overview.png
+    :width: 100%
+    :alt: OpenZaak Componenten-overzicht
+
+Implementatie
+=========
 
-.. _`referentie implementaties van VNG Realisatie`: https://github.com/VNG-Realisatie/gemma-zaken
+Deze repository bevat de broncode voor de API's. Om gebruik te kunnen maken van de API's moeten deze ergens gehost worden als een service. Als onderdeel van de ontwikkelstraat worden bij elke nieuwe versie van OpenZaak containers voor Docker gecreëerd die direct kunnen worden uitgerold in een Kubernetes cluster. Ook wordt er een applicatie VM image gecreëerd. In een diagram ziet de hosting van in een Kubernetes cluster als volgt uit:
+
+.. image:: assets/openzaak-deployment.png
+    :width: 100%
+    :alt: OpenZaak Deployment
 
 Links
 =====
 
+* `Standaard`_
 * `Documentatie`_
 * `Docker Hub`_
 
 .. _`Documentatie`: https://open-zaak.readthedocs.io/en/latest/
 .. _`Docker Hub`: https://hub.docker.com/u/openzaak
+.. _`Standaard`: https://github.com/VNG-Realisatie/gemma-zaken
+
+Bouw
+========
+
+Deze API's zijn ontwikkeld door `Maykin Media B.V.`_ in opdracht van Amsterdam, Rotterdam, Utrecht, Tilburg, Arnhem, Haarlem, 's-Hertogenbosch, Delft en Hoorn, onder regie van `Dimpact`_.
```

Comment:
Even opletten: in 1.0.1 is Hoorn vervangen met een groep gemeenten. Die groep is correct.